### PR TITLE
Callbacks - remove F64 conversion (insolation) and remove unused config

### LINKED ISSUE #2300: Remove Float64 conversion in insolation
Currently, we convert the argument to Float64 when calculating the zenith angle [here](https://github.com/CliMA/ClimaAtmos.jl/blob/d5bc2cff6422dbeef595b70b4853654d79ab5ff1/src/callbacks/callbacks.jl#L118-L119). We shouldn't need this anymore.

### PR DESCRIPTION
**Pull request by akshaysridhar**
Callbacks - remove F64 conversion (insolation)
```
	modified:   src/callbacks/callbacks.jl
	modified:   .buildkite/longruns/pipeline.yml
	modified:   .buildkite/longruns/pipeline.yml
```
Remove unused longrun config
```
	deleted:    longrun_aquaplanet_rhoe_equil_clearsky_highres_hightop_rayleigh35e3_ft32.yml
```
Update ref counter
Closes #2300

### CODE REVIEW

File: config/longrun_configs/longrun_aquaplanet_rhoe_equil_clearsky_highres_hightop_rayleigh35e3_ft32.yml
Comment (szy21) [inline config/longrun_configs/longrun_aquaplanet_rhoe_equil_clearsky_highres_hightop_rayleigh35e3_ft32.yml:1]: Why are we changing this file?

**Review by szy21 — APPROVED** [review]

File: config/longrun_configs/longrun_aquaplanet_rhoe_equil_clearsky_highres_hightop_rayleigh35e3_ft32.yml
Comment (akshaysridhar) [reply config/longrun_configs/longrun_aquaplanet_rhoe_equil_clearsky_highres_hightop_rayleigh35e3_ft32.yml:1]: We don't seem to use this anywhere in the buildkite longrun pipeline - it is the old `50 zelem` configuration - I can revert this change

**Pull request merged**